.github/workflows/release_maven.reusable.workflow.yml

```diff
@@ -0,0 +1,266 @@
+# Copyright (c) 2022 AVI-SPL Inc.
+# All Rights Reserved.
+#
+# Release workflow for Maven projects
+# Update project version, full project build (Maven, docker), Git tagging, deploy to Kubernetes
+# See design document : https://avi-spl.atlassian.net/wiki/spaces/SYM/pages/3011969048/Versioning+strategy
+name: reusable_workflow_release_branch_maven
+
+on:
+  # Reusable workflow
+  workflow_call:
+    inputs:
+      git-ref:
+        description: Project's reference branch (or tag) to be used. E.g. 'release/5.7'
+        required: true
+        type: string
+      pom-xml-file:
+        description: 'The path of the project pom.xml'
+        required: true
+        default: 'pom.xml'
+        type: string
+      new-project-version:
+        description: |
+          (optional) The project version to be set in pom.xml and release.
+          Format : "5.7.0.2206241630"
+          When not specified, version is read from pom.xml and a timestamp is generated
+        required: false
+        type: string
+      maven-goals:
+        description: 'The goals for the mvn command (optional)'
+        required: false
+        type: string
+        default: 'clean deploy'
+      maven-other-options:
+        description: 'Maven options added at the end of the mvn command, overrides previous options'
```

Comment:
so, if default from previous option is removed, user has to add full command here?
Not sure why we don't force this one to be`mvn clean deploy -DskipTests=false -DskipITs=false` and user does not need to specify anything...maybe there is a reasoning behind these 2 steps that I'm not aware of?

Reply:
Hello,
I don't understand your question, there is a default value for each of those parameters:
![image](https://user-images.githubusercontent.com/91557383/183831030-62e53e32-f05d-45e6-b16c-9552a602a129.png)